containerfile/oc-client

```diff
@@ -0,0 +1,8 @@
+FROM quay.io/appuio/oc:v4.16
```

Comment:
Let's use Red Hat shipped interfaces. The purpose of this repo is very much in line with use of Red Hat products, not pure upstream.